src/utils/checkDate.tsx

```diff
@@ -0,0 +1,8 @@
+import moment from 'moment';
+
+export default function checkDate(date: string): number {
```

Comment:
Is there a better name for this function that more specifically says what it returns?